make/photon/prepare/templates/chartserver/env.jinja

```diff
@@ -1,5 +1,12 @@
 ## Settings should be set
+{%if internal_tls.enabled %}
+PORT=9443
+TLS_CERT=/etc/harbor/ssl/chartmuseum.crt
```

Comment:
Have you checked the helm chart?  If the paths to the cert and keys are hardcoded would it be easier for user to enable tls on chart?